src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -1380,6 +1384,69 @@ GUI_TEST_CLASS_DEFINITION(test_7451) {
     GTUtilsStartPage::checkRecentListUrl(os, "test_7451.fa", false);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7455) {
+    // 1. Open "_common_data/regression/7455/clipboard.gb"
+    GTFileDialog::openFile(os, testDir + "_common_data/regression/7455/clipboard.gb");
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
+
+    // 2. Open the "Find restriction sites" dialog, choose "AaaI" (vary first one) only and click OK.
+    GTUtilsDialog::waitForDialog(os, new FindEnzymesDialogFiller(os, {"AaaI"}));
+    GTWidget::click(os, GTWidget::findWidget(os, "Find restriction sites_widget"));
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    // 3. Right click -> Cloning -> Digest into fragments...
+    class DigestScenario : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus& os) override {
+            // 4. Select "AaaI" and click "Add---->"
+            // 5. Go to the "Conserved annotations" tab
+            QWidget* dialog = GTWidget::getActiveModalWidget(os);
+            QListWidget* availableEnzymeWidget = GTWidget::findExactWidget<QListWidget*>(os, "availableEnzymeWidget", dialog);
+            CHECK_SET_ERR(availableEnzymeWidget != nullptr, "Cannot find available enzyme list widget");
+
+            QList<QListWidgetItem*> items = availableEnzymeWidget->findItems("AaaI : 2 cut(s)", Qt::MatchExactly);
+            CHECK_SET_ERR(items.size() == 1, "Unexpected number of enzymes");
+
+            GTListWidget::click(os, availableEnzymeWidget, "AaaI : 2 cut(s)");
+            GTWidget::click(os, GTWidget::findWidget(os, "addButton", dialog));
+            GTTabWidget::clickTab(os, "tabWidget", dialog, 1);
+
+
+            class SelectAnnotationScenario : public CustomScenario {
+            public:
+                void run(HI::GUITestOpStatus& os) override {
+                    // 6. Click "Add", choose the only option and click "OK".
+                    QWidget* dialog = GTWidget::getActiveModalWidget(os);
+                    auto selectAnnotationsList = GTWidget::findWidgetByType<QListWidget*>(os, dialog, "Cant find the \"Select annotations\" list");
+                    auto items = GTListWidget::getItems(os, selectAnnotationsList);
+                    CHECK_SET_ERR(items.size() == 1, "Unexpected number of annotations");
+
+                    GTListWidget::click(os, selectAnnotationsList, items.first());
+                    GTUtilsDialog::clickButtonBox(os, dialog, QDialogButtonBox::Ok);
+                }
+            };
+
+            GTUtilsDialog::waitForDialog(os, new Filler(os, "select_annotations_dlalog", new SelectAnnotationScenario));
```

Comment:
Use () for constructors (the common style in UGENE)

Reply:
OK